File: q1.py
"""
Write a program that accepts a positive integer n and prints an n × n
multiplication table.
For example, if n = 4, the output should be:
1 2 3 4
2 4 6 8
3 6 9 12
4 8 12 16
"""

n = int(input("Enter n: "))

for i in range(1,n+1):
    for j in range(1,n+1):
        print(i*j,end = " ")
    print("\n")

File: 23-09-2026/lab6/q1.c
#include<stdio.h>
#include<math.h>

/*
 A vibration sensor records the velocity of a machine component at regular intervals.
The readings may be positive or negative depending on the direction of motion.
Engineers use the root mean square (RMS) value to measure the overall strength of the vibration.

For n readings a[0], a[1], ..., a[n-1], the RMS value is

RMS = sqrt((a[0]^2 + a[1]^2 + ... + a[n-1]^2) / n)

Write a function

double rms(double a[], int n);

that accepts an array of vibration readings and its length and returns the RMS value.

The array is indexed starting from 0.

Input
The first line contains an integer n, the number of readings.
The second line contains n real numbers representing the vibration readings.

Output
Print the RMS value rounded to two decimal places.*/

double rms(double a[], int n){
	double result;
	for(int i =0;i<n;i++){
		result+=pow(a[i],2);
	}
	result = sqrt(result/n);
	return result;
}

int main(){
	int n;
	scanf("%d",&n);
	double a[n];
	for(int i=0;i<n;i++){
		scanf("%lf",&a[i]);
	}
	printf("%.2lf",rms(a, n));
	return 0;
	}


	

